client/context/query.go

```diff
@@ -134,20 +144,20 @@ func (ctx CLIContext) queryAccount(addr sdk.AccAddress) ([]byte, error) {
 
 	route := fmt.Sprintf("custom/%s/%s", ctx.AccountStore, authtypes.QueryAccount)
 
-	res, err := ctx.QueryWithData(route, bz)
+	resp, err := ctx.query(route, bz)
 	if err != nil {
 		return nil, err
 	}
 
-	return res, nil
+	return resp.Value, nil
 }
 
 // query performs a query from a Tendermint node with the provided store name
 // and path.
-func (ctx CLIContext) query(path string, key cmn.HexBytes) (res []byte, err error) {
+func (ctx CLIContext) query(path string, key cmn.HexBytes) (resp abci.ResponseQuery, err error) {
```

Comment:
maybe this should return a height too for consistency?

Reply:
this would be good

Reply:
maybe just return `[]byte, int64, err` unless there would be value for other queries to use something else in the [ResponseQuery](https://github.com/tendermint/tendermint/blob/4253e67c07c69be6d7f7263ab03944ce30a9fc90/abci/types/types.pb.go#L1879)?

Reply:
Also should I update `GetAccount` and `GetAccountSequence` to also return the height for consistency?

Reply:
Yes for the 1st question -- see my other response; we don't even need `QueryStoreWithHeight`!

As for `GetAccount` and `GetAccountSequence` I would say no. Ideally, we even remove those since we have queriers for `x/auth`. Maybe a follow-up PR?